cli/internal/codegen/client.go

```diff
@@ -44,7 +45,7 @@ func Detect(path string) (lang Lang, ok bool) {
 }
 
 // Client generates an API client based on the given app metadata.
-func Client(lang Lang, appSlug string, md *meta.Data) (code []byte, err error) {
+func Client(lang Lang, appSlug string, md *meta.Data, tsOptions *daemonpb.GenClientRequest_TypeScriptOptions) (code []byte, err error) {
```

Comment:
I guess making the typescript options part of this signature is intended as a short-term thing? Ideally a nicer interface would be nicer to represent `lang+options` as a single protobuf message with a `oneof`